analytics/filesystem/testFiles/test-20200303

```diff
@@ -0,0 +1,5 @@
+2020/03/03 13:53:18 DEBUG {"type":"/openrtb2/auction","Status":0,"Errors":null,"Request":null,"Response":null}
```

Comment:
Should these testFiles be committed?

Reply:
No that was a mistake, but shouldnt the gitignore file ignore the analytics dir?

Reply:
I would expect it to, yes. Doesn't seem to be though. I haven't seen these accumulate locally for me. I guess the cleanup part of the test failed or was skipped during a debug session? In any event, I agree that should be in gitignore. If you can through that in here, great. If not, no worries and I'll throw it in my next PR.